src/app/pages/communication/components/privacy-notice/__tests__/privacy-notice.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { ComponentFixture, async, TestBed } from '@angular/core/testing';
+import { IonicModule, Config } from '@ionic/angular';
+import { TranslateModule, TranslateService } from '@ngx-translate/core';
+import { ConfigMock } from 'ionic-mocks';
+import { configureTestSuite } from 'ng-bullet';
+import { translateServiceMock } from '@shared/mocks/translate';
+import { PrivacyNoticeComponent } from '../privacy-notice';
+
+describe('PrivacyNoticeComponent', () => {
+  let fixture: ComponentFixture<PrivacyNoticeComponent>;
+  let component: PrivacyNoticeComponent;
+
+  configureTestSuite(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        PrivacyNoticeComponent,
+      ],
+      imports: [
+        IonicModule,
+        TranslateModule,
+      ],
+      providers: [
+        { provide: TranslateService, useValue: translateServiceMock },
+        { provide: Config, useFactory: () => ConfigMock.instance() },
```

Comment:
Don't think we need this Config declaration

Reply:
funny thing is, i picked matt up for that yesterday. removed